tools/mason/MasonArgParse.chpl

```diff
@@ -766,17 +757,30 @@ module MasonArgParse {
                                 + "as default values at this time");
       }
 
-      var action = new owned Option(name=name,
+      var handler = new owned Option(name=name,
                                     numOpts=opts.size,
                                     opts=opts,
-                                    numArgs=numArgs,
+                                    numArgs=nArgs,
                                     required=required,
                                     defaultValue=myDefault
                                     );
-      return _addAction(action);
+      return _addHandler(handler);
     }
   }
 
+  // helper to prepare numArgs ranges for use
+  proc _prepareRange(rIn: range(?)) :range throws {
```

Comment:
Let's put a space after the colon in the return type. (I'm ok with the space before it being present, or not).